test/lib/sstable_test_env.hh

```diff
@@ -91,6 +95,11 @@ public:
         return reusable_sst(std::move(schema), dir, generation).then([] (auto ptr) { return make_ready_future<>(); });
     }
 
+
+    compaction_manager& get_compaction_manager() noexcept {
+        return *_compaction_manager;
```

Comment:
Wait, wasn't the definition of this attribute `compaction_manager _compaction manager`? In this case this code should not compile

Reply:
oops, the method is not used until 3628e26e86 where it's fixed.
I can remove it here entirely until the patch that introduces its users.